back/test/integration/TestTechMapsApi.js

```diff
@@ -63,26 +62,331 @@ describe("API", () => {
       expect(res).to.have.status(404);
     });
 
-    it.skip("should bump up version when modified techmap put", async() => {
+    it("should bump up version when modified techmap put", async () => {
       // todo: make sure all fields are checked.
```

Comment:
// todo: make sure all fields are checked.

In general we should avoid comitting todos. Remove and address.